### DISCUSSION TITLE
Segment.com integration

### DISCUSSION THREAD

**Discussion by metmarkosaric** [plausible/analytics#122] [Anything Else]
"Should be easy to integrate for Segment users"

**Comment by metmarkosaric** [comment]
Just got a message that as Segment integrates with Google Tag Manager, it is also possible to integrate Plausible with Segment using Google Tag Manager. Our instructions for GTM are here https://plausible.io/docs/google-tag-manager

**Reply by amosbastian** [reply]
Are there any plans to work on this for people who don't want to use GTM?

**Comment by msakbar** [comment]
+1 on this request, is this something that's on the roadmap?

**Reply by metmarkosaric** [reply]
hi @msakbar! this is something we'd like to build in the future but no short-term plans to start working on it. we have several big projects ahead of us such as custom dimensions for pageviews, comparisons and funnels that are more frequently requested. when we have an update on this, we'll post it here. thanks!

**Reply by msakbar** [reply]
hey, thanks for getting back to me on this!
